interactions/client/models/utils.py

```diff
@@ -50,7 +61,16 @@ async def deferring_func(ctx: Union["CommandContext", "ComponentContext"], *args
                 loop = get_running_loop()
             except RuntimeError as e:
                 raise RuntimeError("No running event loop detected!") from e
-            task: Task = loop.create_task(coro(ctx, *args, **kwargs))
+
+            if "self" in getfullargspec(coro).args:
```

Comment:
This method isn't reliable since users may name the `self` parameter something else.
Could you try using the qualname trick?
Iirc, you check if the dunder qualname (of the `coro` or of the `coro.__func__`, i forgot) has a "." in it.
I think Astrea used it in molter

Reply:
I think I will make autodefer run completely different

Reply:
nvm.